code/modules/mining/mine_items.dm

```diff
@@ -175,6 +179,12 @@
 	throwforce = 7
 	w_class = ITEM_SIZE_SMALL
 
+/obj/item/shovel/spade/Initialize(ml, material_key)
```

Comment:
Would be preferable to have a var on /shovel containing the tool quality, pass that in on /shovel Initialize, then change it on /spade.

Reply:
I can do that.